data/entities.json5

```diff
@@ -824,12 +833,16 @@
     "categories": ["social"],
     "domains": [
       "platform.linkedin.com",
-      "snap.licdn.com",
       "bizographics.com",
       "slideshare.com",
       "slidesharecdn.com"
     ]
   },
+  {
+    "name": "LinkedIn Ads",
+    "categories": ["ad"],
+    "domains": ["ads.linkedin.com", "snap.licdn.com", "www.linkedin.com"]
```

Comment:
can you add a comment (yay json5! :D) on the `www.linkedin.com` what sorts of paths are ads?

someday when it's path granularity it'd be nice to know where these came from :)

Reply:
Sweet, I'll add a comment. Here's the request we saw on wired.com...

<img width="1395" alt="Screen Shot 2019-04-30 at 2 13 34 PM" src="https://user-images.githubusercontent.com/204268/56937704-4758c100-6b52-11e9-89f0-130eb18fc43b.png">